app/models/feedback.rb

```diff
@@ -26,12 +26,13 @@ class Feedback < ActiveRecord::Base
     "Caseflow Reader" => "Reader",
     "Caseflow Hearing Prep" => "Hearing Prep",
     "Caseflow Intake" => "Intake",
+    "Caseflow Queue" => "Queue",
     "Caseflow" => "Certification",
```

Comment:
We don't necessarily want to default everything to Certification. Should we change this from ` "Caseflow" => "Certification",` to  `"Caseflow" => "Caseflow",`?